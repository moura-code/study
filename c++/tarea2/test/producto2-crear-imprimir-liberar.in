crearFecha 1/1/2001
crearProducto 7 producto1 15
idProducto
imprimirProducto
liberarProducto
crearFecha 2/7/2003
crearProducto 3 producto2 20
imprimirProducto
idProducto
liberarProducto
Fin
