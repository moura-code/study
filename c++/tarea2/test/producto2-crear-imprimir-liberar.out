Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Producto creado en forma exitosa.
3> ID: 7
4> Producto: 7
producto1
Precio: 15
Ingresado el: 1/1/2001
5> Producto liberado en forma exitosa.
6> Fecha creada en forma exitosa.
7> Producto creado en forma exitosa.
8> Producto: 3
producto2
Precio: 20
Ingresado el: 2/7/2003
9> ID: 3
10> Producto liberado en forma exitosa.
11> Fin.
